main.go

```diff
@@ -10,6 +10,7 @@ import (
 	istioinformers "istio.io/client-go/pkg/informers/externalversions"
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
Needed for us to be able to run the code using the Azure KUBECONFIGs. I wonder if this is needed in the AKS-Engine build?

Reply:
I want to say it's already there since it knows to prompt for the authentication code.

Reply:
Hmm... that's fair.